pkg/api/v1/types.go

```diff
@@ -1478,6 +1484,14 @@ type Container struct {
 	// Default is false.
 	// +optional
 	TTY bool `json:"tty,omitempty" protobuf:"varint,18,opt,name=tty"`
+	// List of sources to populate environment variables in the container.
+	// The keys defined within a source must be a C_IDENTIFIER.
+	// When a key exists in multiple sources, the value associated with the last
+	// source will take precedence.
+	// All EnvVars will take precedence over any listed source.
```

Comment:
We generate user-facing documentation from this file, so please use json field names and plain English rather than Go type and field names.